xmodule/modulestore/tests/factories.py

```diff
@@ -366,6 +366,7 @@ def _create(cls, target_class, **kwargs):  # lint-amnesty, pylint: disable=argum
         has_score = kwargs.pop('has_score', None)
         submission_start = kwargs.pop('submission_start', None)
         submission_end = kwargs.pop('submission_end', None)
+        is_proctored_exam = kwargs.pop('is_proctored_exam', None)
```

Comment:
I think this line is hiding an underlying issue (see my comment in test_exams.py). because is_proctored_exam is never actually being saved from what I can tell and causing some false positives in proctoring related tests.

Reply:
Updated